mk/Common.mk

```diff
@@ -232,128 +240,99 @@ endif
 endif
 
 ###############################################################################
-# Imperas Instruction Set Simulator
+# Determine the values of the CV_SW_ variables.
+# The priority order is ENV > TEST > CFG.
 
-DV_OVPM_HOME    = $(CORE_V_VERIF)/vendor_lib/imperas
-DV_OVPM_MODEL   = $(DV_OVPM_HOME)/imperas_DV_COREV
-DV_OVPM_DESIGN  = $(DV_OVPM_HOME)/design
-OVP_MODEL_DPI   = $(DV_OVPM_MODEL)/bin/Linux64/imperas_CV32.dpi.so
-#OVP_CTRL_FILE   = $(DV_OVPM_DESIGN)/riscv_CV32E40P.ic
+ifndef CV_SW_TOOLCHAIN
+ifdef  TEST_CV_SW_TOOLCHAIN
```

Comment:
This may be a bit pedantic, but for more robustness we may also want to check for an empty, but defined variable. I know it is bad practice to set variables empty when they are not supposed to exist, but this happens.  (e.g. `export CV_SW_TOOLCHAIN=""`  is not the same as  `unsetenv CV_SW_TOOLCHAIN` )

Reply:
You are right, these are different.  The good news is that the current checks that are in place cannot distinguish between `CV_SW_TOOLCHAIN=""` is not the same as `unsetenv CV_SW_TOOLCHAIN`.  I have tried both and each time receive the follow error message:
```
Common.mk:253: *** CV_SW_TOOLCHAIN not defined in either the shell environment, test.yaml or cfg.yaml.  Stop.
```